ssg/oval_object_model/oval_container.py

```diff
@@ -71,14 +71,14 @@ def _copy_component(destination, source_of_components):
         add_oval_component(component, destination)
 
 
-def _remove_keys_from_dict(to_remove, dict_):
+def _remove_keys_from_dict(dict_, to_remove):
```

Comment:
Yep, it is a bug.

Reply:
??

Reply:
In the original form the function `_remove_keys_from_dict` can't be used in `_call_function_for_every_component` unlike its counterpart.